tests/validation/tests/v1_api/test_monitoring_v2.py

```diff
@@ -0,0 +1,142 @@
+from .common import *  # NOQA
+import pytest
+import requests
+import semver
+
+namespace = {'client': None,
+             'cluster': None,
+             'rancher_catalog': None,
+             'project': None}
+monitoring_v2 = 'rancher-monitoring'
```

Comment:
is this the chart name? if so can we add an apt variable name  ?

Reply:
fixed. It is now `m_chart_name`